__tests__/integration/browser/browser_exec.test.ts

```diff
@@ -0,0 +1,47 @@
+import type { ClickHouseClient } from '@clickhouse/client-common'
+import { createTestClient } from '../../utils'
+import { getAsText } from '../../../packages/client-browser/src/utils' // FIXME: Karma does not like "proper" typescript path here
```

Comment:
another reason to keep the test in the appropriate package

Reply:
This is a good one